staging/src/k8s.io/apiserver/pkg/util/logs/logs.go

```diff
@@ -69,8 +69,12 @@ func NewLogger(prefix string) *log.Logger {
 	return log.New(GlogWriter{}, prefix, 0)
 }
 
-// GlogSetter is a setter to set glog level.
-func GlogSetter(val string) (string, error) {
+// GlogSetterOrGetter is to used to set glog level or get glog level .
+func GlogSetterOrGetter(val string) (string, error) {
+	if len(val) == 0 {
+		currentLevel := flag.Lookup("v").Value.(flag.Getter).Get().(glog.Level)
```

Comment:
We can only get log level set by flag. But not that set by this handler. So I think it make no sense.